src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx

```diff
@@ -1,13 +1,213 @@
-import { ArtQuizMain } from "Apps/ArtQuiz/Components/ArtQuizMain"
-import { MetaTags } from "Components/MetaTags"
-import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtQuizArtworks_quiz$data } from "__generated__/ArtQuizArtworks_quiz.graphql"
+import { AuthContextModule } from "@artsy/cohesion"
+import {
+  ArrowLeftIcon,
+  Box,
+  Clickable,
+  Image,
+  Flex,
+  FullBleed,
+  Text,
+} from "@artsy/palette"
+import { ArtQuizButton } from "Apps/ArtQuiz/Components/ArtQuizButton"
+import {
+  ArtQuizCard,
+  Mode,
+  useArtQuizCards,
+} from "Apps/ArtQuiz/Components/ArtQuizCard"
+import { useSwipe } from "Apps/ArtQuiz/Hooks/useSwipe"
+import { useDislikeArtwork } from "Apps/ArtQuiz/Hooks/useDislikeArtwork"
+import { useSaveArtwork } from "Components/Artwork/SaveButton/useSaveArtwork"
+import { FC, useCallback } from "react"
+import { RouterLink } from "System/Router/RouterLink"
+import { useRouter } from "System/Router/useRouter"
+
+interface ArtQuizArtworksProps {
+  quiz: ArtQuizArtworks_quiz$data
+}
+
+export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ quiz }) => {
+  const { router } = useRouter()
+  const resume = quiz.quizArtworkConnection!.edges!.findIndex(edge => {
+    if (edge && "interactedAt" in edge) {
+      return edge.interactedAt === null
+    }
+    return false
```

Comment:
You should be able to write this as a single condition rather than an if statement